Generator/Generator/BuiltinGen.swift

```diff
@@ -813,6 +812,14 @@ func generateBuiltinClasses (values: [JGodotBuiltinClass], outputDir: String?) a
             generateBuiltinMethods(p, bc, bc.methods ?? [], typeName, typeEnum, isStruct: kind == .isStruct)
             generateBuiltinOperators (p, bc, typeName: typeName)
             generateBuiltinConstants (p, bc, typeName: typeName)
+                        
+            p("public func toVariant() -> Variant") {
```

Comment:
For new public API entry points, could we include the relevant documentation?

I think this would do:

/// Wraps this type into a Variant